src/util/handlebars/handlebar-helpers.ts

```diff
@@ -27,8 +27,7 @@ export const registerHandlebarsHelpers = () => {
 				: `${numberOfSyncedRepos} / ${totalNumberOfRepos}`
 	);
 
-	//check to display repo count as zero in case of org with zero repos
-	hbs.registerHelper("hasTotalNumberOfRepos", (totalNumberOfRepos: number) => (totalNumberOfRepos || totalNumberOfRepos === 0));
+	hbs.registerHelper("checkRepoCount", (totalNumberOfRepos: number) => (totalNumberOfRepos >= 0));
```

Comment:
Wondering if  `typeof totalNumberOfRepos === 'number'` would make it more clear?...

Reply:
yes it would !

Reply:
no i thought applying that as an additional check type check and no to be equal or grater than zero

Reply:
`typeof totalNumberOfRepos === 'number' && totalNumberOfRepos >=0` looks good to my taste :)

Reply:
We already have typescript check for that, so feels redundant